wasm2map/src/lib.rs

```diff
@@ -454,4 +485,86 @@ impl WASM {
         result.push(n as u8);
         result
     }
+
+    // Inspired by:
+    // <https://github.com/serde-rs/json/blob/a0ddb25ff6b86f43912f8fc637797bcbb920c61e/src/ser.rs#L1997-L2063>
+    fn json_encode(string: &str) -> Cow<'_, str> {
```

Comment:
Since JSON encoding is not WASM-specific, I think this deserves a new mod in a separate source file (i.e. json.rs or similar). In fact, the VLQ encoding falls into the same category, but that can be moved later.

Reply:
Done.
I'm happy to move the VLQ encoding as well, just say the word!

Reply:
Sure, if you feel like it! It definitely looks cleaner that way.

Reply:
Done.